build/templates/misc-template.mjs

```diff
@@ -1,7 +1,7 @@
 const miscTemplate = ( { componentName, jsx }, { tpl }, ) => tpl`
 import React from "react";
 
-const ${componentName} = ({size=24, ...props}) => {
+const ${componentName} = ({width=24, height=24, ...props}) => {
```

Comment:
@nitinprojects _a @nitinprojects _a This is a breaking change. We are already using many misc icons in multiple Neeto products with the size prop.

As per the new approach, the size prop has been replaced with width and height. If a misc icon is used with a size prop of 16px, it won’t work as expected. The icon will render at 24px width and height, which may break the UI in some places.

I think we can revert this change and continue using the size prop like before. If needed, we can override the width or height using Tailwind utility classes, since they apply with the !important rule by default.

<img width="1407" alt="Screenshot 2025-04-04 at 11 23 57 AM" src="https://github.com/user-attachments/assets/57e09f14-7839-44de-b99e-8b42e5cf2275" />